### PR TITLE
Add aestream

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge/help-python-c this is ready for review :)

File: recipes/aestream/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  sha256: eeaa9dc49465bf25de80a549b78ee767cf3b34c1047caeb42be5e089d003d0ad
+  patches:
+    - unvendor.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  ignore_run_exports:
+    # cmake cannot find boost without libboost-devel but it is not needed at runtime
+    - libboost  # [linux]
```
Comment (xhochy) [inline recipes/aestream/meta.yaml:19]:
```suggestion
  ignore_run_exports_from:
    # cmake cannot find boost without libboost-devel but it is not needed at runtime
    - libboost-devel  # [linux]
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/aestream/conda_build_config.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+macos_min_version:             # [osx and x86_64]
```
[the remark this answers is not in the document]
Comment (Tobias-Fischer) [reply recipes/aestream/conda_build_config.yaml:1]: Yes:
`/Users/runner/Miniforge3/conda-bld/aestream_1710738893777/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.10/site-packages/nanobind/src/nb_type.cpp:231:13: error: aligned deallocation function of type 'void (void *, std::align_val_t) noexcept' is only available on macOS 10.13 or newer
            operator delete(p, std::align_val_t(t->align));
            ^`

see https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=897985&view=logs&jobId=e35d4f76-8ff2-5536-d795-df91e63eb9f7&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310

**Comment by Tobias-Fischer** [comment]
Thanks @xhochy for your review, I think I’ve addressed your comments :)

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by Tobias-Fischer** [comment]
All done, thanks @xhochy

**Review by xhochy — APPROVED** [review]

**Comment by Tobias-Fischer** [comment]
Many thanks @xhochy

**Pull request merged**